tests/test_plugin_features.py

```diff
@@ -476,6 +476,34 @@ def test_pf_with_no_feature_label():
     assert label == "Plugin feature {}".format(feature_name)
 
 
+def test_pf_with_user_config_section():
+    """Use a plugin feature with the user defined config section"""
+    # setup a plugin method that uses user config section
+    def compute_with_user_section(rtdc_ds):
+        if rtdc_ds.config["user"]["channel"]:
```

Comment:
Here you should also check whether it `isinstance` of `bool`, because ``bool("true") == True``.